docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>Are we jumping right to PEA? What about gate folding ZNE? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
> Are we jumping right to PEA? What about gate folding ZNE?

Are you asking us to add another experiment that uses gate folding ZNE, or am I just missing some explanatory text?

Reply:
The snippet I posted [here](https://github.com/Qiskit/documentation/pull/2223#discussion_r1831689014) had 

> - noisy expectation value 0.2, zero-noise extrapolation with gate folding

Jumping directly from nothing to PEA is a bit drastic, since PEA is much more expensive than ZNE.